exo-setup/src/docker-setup.ls

```diff
@@ -68,26 +68,34 @@ class DockerSetup
 
   # builds the Docker config for a service dependency
   _get-service-dependency-docker-config: (dependency-name, dependency-config) ->
-    if dependency-config.volumes
-      data-path = global-exosphere-directory @app-config.name, dependency-name
-      fs.ensure-dir-sync data-path
-      rendered-volumes =  map ((volume) -> Handlebars.compile(volume)({"EXO_DATA_PATH": data-path})), dependency-config.volumes
-
     Obj.compact do
       image: "#{dependency-config.image}:#{dependency-config.version}"
       container_name: dependency-name + dependency-config.version
       ports: dependency-config.ports
-      volumes: rendered-volumes or undefined
-
-
-  _get-external-service-docker-config: ->
-    | !@docker-image => throw new Error red "No location or docker-image specified"
-    docker-config = {}
-    docker-config[@role] =
-      image: @docker-image
-      container_name: @role
-      depends_on: @_get-app-dependencies!
-    docker-config
+      volumes: @_get-rendered-volumes dependency-config.volumes, dependency-name
+
+
+  _get-external-service-docker-config: (done) ~>
+    | !@docker-image => done new Error red "No location or docker-image specified"
+    DockerHelper.cat-file image: @docker-image, file-name: 'service.yml', (err, external-service-config) ~>
+      | err => done new Error red "Could not find the configuration for the docker-image"
```

Comment:
Let's just bubble up the error that's returned by `DockerHelper.cat-file`, since we don't know why it failed for sure.
```
| err => done err
```